components.tsx

```diff
@@ -116,18 +116,22 @@ const AppText = (props: {
 const Button = (props: {
   text: string;
   onPress: () => void;
-  secondary?: boolean;
+  variant?: "primary" | "secondary";
 }) => (
   <View>
     <TouchableOpacity
-      style={props.secondary ? styles.secondaryButtonStyle : styles.buttonStyle}
+      style={
+        props.variant === "secondary"
+          ? styles.secondaryButtonStyle
+          : styles.buttonStyle
+      }
```

Comment:
if you create your styles like:
```
const styles = StyleSheet.create({
  primaryButton: {...},
  secondaryButton: {...}
});
```
you could write:
```
style={styles[`${props.variant || 'primary'}Button`}
```
which scales better

Reply:
oooooh!

Reply:
jeg liker virkelig denne metoden. Må huske den